sematic/db/migrations/20221208183307_create_server_settings_file.py

```diff
@@ -0,0 +1,21 @@
+# Standard Library
+import os
+import shutil
+
+# Sematic
+from sematic.config.server_settings import get_server_settings_scope
+from sematic.config.user_settings import get_user_settings_scope
+
+
+def up():
+    server_settings_file_path = get_server_settings_scope().settings_file_path
+    user_settings_file_path = get_user_settings_scope().settings_file_path
+
+    if os.path.isfile(user_settings_file_path) and not os.path.isfile(
+        server_settings_file_path
+    ):
+        shutil.copy(user_settings_file_path, server_settings_file_path)
```

Comment:
Please also do a load and save of each file, so that settings that belong to the other file will be pruned out.